src/ApiService/ApiService/OneFuzzTypes/Metrics.cs

```diff
@@ -0,0 +1,376 @@
+﻿using System.Reflection;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
+
+namespace Microsoft.OneFuzz.Service;
+
+/// <summary>
+/// Identifies the enum type associated with the metric class
+/// </summary>
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
+public class MetricTypeAttribute : Attribute {
+    public MetricTypeAttribute(MetricType metricType) {
+        this.MetricType = metricType;
+    }
+
+    public MetricType MetricType { get; }
+}
+
+
+public enum MetricType {
```

Comment:
Same comment as the other PR. Instead of creating a new type of metrics. We can just reuse the Event and modify the Event type as needed.

Reply:
I'm conflicted about this. I don't like the ambiguity of records called 'events' being used as metrics when we log both. I think it makes sense from an organizational perspective to keep them separate. 

A middle ground could be removing the 'metric types that I haven't yet used and only create them on a case by case basis as needed.